tests/schema_int.rs

```diff
@@ -0,0 +1,1094 @@
+use std::{fs::File, io::BufReader, path::PathBuf, str::FromStr};
```

Comment:
I think I missed why this test file is called `schema_int.rs`. It might be a stupid question, but I would appreciate if you could explain it to me

Reply:
There was on master a schema.rs as a integration test, so I made it a new target just to make it clear what changes.

It could be merged with `schema.rs`

Reply:
I see, that was int for integration. Thanks for separating it so to make the review easier to read. I think that at the end it would be best to merge it in `test/schema.rs`, yes.